### DISCUSSION TITLE
死二：这里居然没有对空

### DISCUSSION THREAD

**Discussion by 123-Invincibleswordsman** [share121/inter-knot#1141] [其他]
忆昔当年泪不干
![19142120dc275JfFQKzIoDnskww](https://github.com/user-attachments/assets/04aa10cf-cc4f-4fb8-97c9-1bf3b105e0ea)

**Comment by lock-key-H** [comment]
这里也有村里人！！

**Comment by demolute** [comment]
冷知识，崩坏三以前也没有

**Comment by OttoApocalypse01** [comment]
但是这里有身后判定<img src="https://bbs-static.miyoushe.com/static/2023/10/18/f11a60bb7e27541dd6856c0c32107a55_2162956347575324575.png" alt="那维莱特-下雨了" width="75" >

**Reply by OttoApocalypse01** [reply]
是仙舟罗浮云骑军的彦青小朋友吗

**Reply by demolute** [reply]
我抄，盒！
